hotels/urls.py

```diff
@@ -0,0 +1,8 @@
+from django.urls import path
+
+from . import views
+
+urlpatterns = [
+    path("", views.HotelListViwe.as_view()),
+    path("/<int:pk>", views.HotelDetailViwe.as_view()),
```

Comment:
views모델을 한 번에 import 하기보다는 모듈 내에 있는 클래스를 정확히 지칭해서 import 하시는 것을 추천드립니다. 불필요한 메모리 사용을 줄일 수 있습니다.

```suggestion
from .vies import (
    HotelListView,
    HotelDetailView
)
urlpatterns = [
    path("", HotelListViwe.as_view()),
    path("/<int:pk>", HotelDetailViwe.as_view()),
```